### ISSUE TITLE
"go get github.com/vbauerster/mpb" yields warning message

### ISSUE THREAD

**Issue by retorquere** [vbauerster/mpb#1]
When I "go get" this package, I get this complaint:

```
package context: unrecognized import path "context" (import path does not begin with hostname)
```

I am guessing this must be due to `progress.go` and `bar.go` importing `context` while there is no `context.go`.

**Comment by vbauerster** [comment]
What version of go do you use? [context](https://golang.org/doc/go1.7#context) was included to the standard lib, since version 1.7. So most probably you did "go get" with version lower than 1.7

**Comment by retorquere** [comment]
Ah, I did. I'm on 1.6. I'll see about upgrading.
